backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_Order.java

```diff
@@ -635,4 +637,80 @@ private void setDefaultBillToBPartnerLocation(@NonNull final I_C_Order order)
 			order.setC_BPartner_Location_ID(billToLocation.getC_BPartner_Location_ID());
 		}
 	}
+
+
+	@ModelChange(timings = { ModelValidator.TYPE_AFTER_CHANGE },
+			ifColumnsChanged = {
+					I_C_Order.COLUMNNAME_UserElementString1,
+					I_C_Order.COLUMNNAME_UserElementString2,
+					I_C_Order.COLUMNNAME_UserElementString3,
+					I_C_Order.COLUMNNAME_UserElementString4,
+					I_C_Order.COLUMNNAME_UserElementString5,
+					I_C_Order.COLUMNNAME_UserElementString6,
+					I_C_Order.COLUMNNAME_UserElementString7,
+			})
+	public void copyDimensionToLines(@NonNull final I_C_Order order)
+	{
+		final List<I_C_OrderLine> orderLines = orderDAO.retrieveOrderLines(OrderId.ofRepoId(order.getC_Order_ID()));
+		if (orderLines.isEmpty())
+		{
+			return;
+		}
+
+		final boolean userElementString1Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString1);
+		final boolean userElementString2Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString2);
+		final boolean userElementString3Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString3);
+		final boolean userElementString4Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString4);
+		final boolean userElementString5Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString5);
+		final boolean userElementString6Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString6);
+		final boolean userElementString7Changed = isValueChanged(order, I_C_Order.COLUMNNAME_UserElementString7);
+
+		for (final I_C_OrderLine line : orderLines)
+		{
+			if (userElementString1Changed)
+			{
+				line.setUserElementString1(order.getUserElementString1());
+			}
+			if (userElementString2Changed)
+			{
+				line.setUserElementString2(order.getUserElementString2());
+			}
+			if (userElementString3Changed)
+			{
+				line.setUserElementString3(order.getUserElementString3());
+			}
+			if (userElementString4Changed)
+			{
+				line.setUserElementString4(order.getUserElementString4());
+			}
+			if (userElementString5Changed)
+			{
+				line.setUserElementString5(order.getUserElementString5());
+			}
+			if (userElementString6Changed)
+			{
+				line.setUserElementString6(order.getUserElementString6());
+			}
+			if (userElementString7Changed)
+			{
+				line.setUserElementString7(order.getUserElementString7());
+			}
+
+			orderBL.save(line);
+		}
+	}
+
+
+	@ModelChange(timings = { ModelValidator.TYPE_AFTER_CHANGE },
+			ifColumnsChanged = { org.compiere.model.I_C_Order.COLUMNNAME_M_SectionCode_ID })
+	@CalloutMethod(columnNames = org.compiere.model.I_C_Order.COLUMNNAME_M_SectionCode_ID)
+	public void updateSectionCode(@NonNull final org.compiere.model.I_C_Order order)
```

Comment:
why not doing it in `copyDimensionToLines` method?

Reply:
right. Merging them.